seg_lapa/config/dataset/lapa.yaml

```diff
@@ -1,6 +1,7 @@
 # @package _group_
+_target_: "config_parse.dataset_conf.LapaConf"
```

Comment:
```suggestion
```
We don't want to use hydra's instantiate functionality anywhere. It'll be replaced with config_parse module.